org.freedesktop.Sdk.Extension.swift.json

```diff
@@ -0,0 +1,206 @@
+{
+    "id": "org.freedesktop.Sdk.Extension.swift",
+    "branch": "23.08",
+    "runtime": "org.freedesktop.Sdk",
+    "build-extension": true,
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.llvm16"
+    ],
+    "runtime-version": "23.08",
+    "separate-locales": false,
+    "appstream-compose": false,
+    "build-options": {
+        "prefix": "/usr/lib/sdk/swift",
+        "prepend-pkg-config-path": "/usr/lib/sdk/swift/lib/pkgconfig/",
+        "prepend-path": "/usr/lib/sdk/swift/bin/:/usr/lib/sdk/llvm14/bin",
+        "prepend-ld-library-path": "/usr/lib/sdk/llvm14/lib:/usr/lib/sdk/swift/lib"
+    },
+    "cleanup": [
+        "/share/info",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "libedit",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.thrysoee.dk/editline/libedit-20221030-3.1.tar.gz",
+                    "sha256": "f0925a5adf4b1bf116ee19766b7daa766917aec198747943b1c4edf67a4be2bb"
+                }
+            ],
+            "cleanup": [
+                "/include",
+                "/lib/pkgconfig",
+                "/share/man"
+            ]
+        },
+        {
+            "name": "swift",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.swift.org/swift-5.9.1-release/centos7/swift-5.9.1-RELEASE/swift-5.9.1-RELEASE-centos7.tar.gz",
```

Comment:
how will this work on aarch64?

Reply:
also you can take something more recent that centos7. The Freedesktop SDK is a pretty recent stack

Reply:
I switched to Ubuntu 22.04, which has an x86_64 and an aarch64 download.

Reply:
I switched once again, this time to the Amazon Linux 2 downloads, because I could remove the "really really dirty" part and the lines that fail on aarch64 that way.